opengapps-packages.mk

```diff
@@ -12,22 +12,28 @@ PRODUCT_PACKAGES += GoogleBackupTransport \
                     SetupWizard \
                     Phonesky \
                     GoogleCalendarSyncAdapter
+                    
+ifneq ($(filter $(call get-allowed-api-levels),23),)
+PRODUCT_PACKAGES += GoogleTTS
+endif
 
 ifneq ($(filter $(TARGET_GAPPS_VARIANT),nano),) # require at least nano
 PRODUCT_PACKAGES += FaceLock \
+                    HotwordEnrollment \
                     Velvet
 
 ifneq ($(filter $(TARGET_GAPPS_VARIANT),micro),) # require at least micro
 PRODUCT_PACKAGES += CalendarGooglePrebuilt \
                     PrebuiltExchange3Google \
                     PrebuiltGmail \
-                    GoogleHome \
-                    GoogleTTS
+                    GoogleHome
+                    
+ifeq ($(filter $(call get-allowed-api-levels),23),)
```

Comment:
This one adds tts if the API level is below 23 so it is not added twice on Marshmallow.